.evergreen/config.yml

```diff
@@ -736,17 +1159,28 @@ axes:
       - id: ubuntu-18.04
         display_name: "Ubuntu 18.04"
         run_on: ubuntu1804-test
+        variables:
+          PYTHON: "/opt/mongodbtoolchain/v3/bin/python"
+          VENV_BIN_DIR: "bin"
       - id: ubuntu-16.04
         display_name: "Ubuntu 16.04"
         run_on: ubuntu1604-test
+        variables:
```

Comment:
We aren't actually using this on 16.04 anywhere in this file, right?

Reply:
nope. I was wondering if we ought to remove this and run everything on 18.04, since currently we have a bit of a mix.

Reply:
I believe that the only existing place we use 18.04 (before this PR) is with the AWS auth tests, as it's required for some of the container tests. We use 16.04 for the rest of the variants because there are no 3.6 binaries for 18.04, and it's easier to just use the same Ubuntu version for all of them.